automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/NotebookInteractionSpec.scala

```diff
@@ -175,5 +175,22 @@ class NotebookInteractionSpec extends FreeSpec with LeonardoTestUtils with Befor
       }
     }
 
+    "should create a notebook with a working Python 3 kernel and import installed packages" in withWebDriver { implicit driver =>
```

Comment:
So we know Hail doesn't work -- but can we add a test to make sure Spark works with Py3?

I've tested using the "pi estimation" example on this page: https://spark.apache.org/examples.html